SAMtags.tex

```diff
@@ -234,13 +239,39 @@ \subsection{Barcodes}
 
 \begin{description}
 \item[BC:Z:\tagvalue{sequence}]
-Barcode sequence, with any quality scores stored in the {\tt QT} tag.
-
-\item[QT:Z:\tagvalue{qualities}]
-Phred quality of the barcode sequence in the {\tt BC} (or {\tt RT}) tag. Same encoding as {\sf QUAL}.
+Barcode sequence (Identifying the sample/library), with any quality scores (optionally) stored in the {\tt QT} tag.
+The {\tt BC} tag should match the {\tt QT} tag in length. 
+In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the barcodes and qualities and places a dash (`{\tt -}') between the barcodes from the same template. 
```

Comment:
This is ambiguous.    It states we "concatenate all the barcodes and qualities" and then "places a dash between the barcodes", but not explicitly "barcodes and qualities".    This could imply that our sequence has separators but not the qualities, but this is at odds with the sentence "the BC tag should match the QT tag in length", unless length here defines the number of nucleotides and not our specific textual encoding.

Also given there are already differences between implementations, perhaps it is better to say:

> The recommended implementation when writing these tags is to place a dash between the barcodes from the same template, and when reading to interpret non-nucleotides or ambiguity codes as a separator.

If we expect the separators to be present in the QT too then we need to explain that the BC tag must be used in conjunction with QT in order to identify which are quality values and which are separators.  If this isn't the case then it should be spelt out explicitly.